src/xcode/ENA/ENA/Source/Scenes/Settings/SettingsViewController.swift

```diff
@@ -134,18 +134,20 @@ final class SettingsViewController: UITableViewController {
 	private func notificationSettings() {
 		let currentCenter = UNUserNotificationCenter.current()
 
-		currentCenter.requestAuthorization(options: [.alert, .sound, .badge]) { [weak self] granted, error in
+		currentCenter.getNotificationSettings { [weak self] (settings) in
```

Comment:
Our current design looks like this if the notifications are not enabled / authorized, yet.
![IMG_7878](https://user-images.githubusercontent.com/15961043/84891435-288df480-b09c-11ea-9325-97b61a5c3557.PNG)

I totally agree with your opinion but since the RKI needs to sign-off changes on the UI, I would recommend closing the current issue (the alert now shows up on the correct screen) and directly create a feature request and discuss that with the designers and the RKI.

Reply:
The popup will not appear here. This is fine.